ui/js/dfv/src/hooks/useValidation.js

```diff
@@ -1,8 +1,29 @@
 import { useState, useEffect } from 'react';
+import {
+	useSelect,
+	useDispatch,
+} from '@wordpress/data';
 
-const useValidation = ( defaultRules = [], value ) => {
+const useValidation = ( defaultRules = [], value, fieldName, strokeKey ) => {
```

Comment:
strokeKey or storeKey?

```suggestion
const useValidation = ( defaultRules = [], value, fieldName, storeKey ) => {
```